content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -227,13 +227,14 @@ local workstation to debug the database that is running in the pod.
 
 与本地 7000 端口建立的连接将转发到运行 Redis 服务器的 pod 的 6379 端口。通过此连接，您可以使用本地工作站来调试在 pod 中运行的数据库。
 
-{{< warning >}}
 <!--
+{{< warning >}}
```

Comment:
This is not a correct fix. Please remove line 231.

The reason is that shortcode like `{{< warning >}}` in a HTML comment still works!!!

Reply:
ok

Reply:
Done!